src/app/components/Home/index.tsx

```diff
@@ -1,10 +1,23 @@
 import React from "react";
+import { Redirect, Route } from "react-router-dom";
+
+import { useSelector } from "react-redux";
+import { getAuthUser } from "../../redux/selectors";
 
 // style
 import { TitleHome } from "./components";
 
-const Home: React.FC = () => (  
-    <TitleHome>This home page ...✨</TitleHome>  
-);
+const Home: React.FC = () => {
+  const user = useSelector(getAuthUser);
+  return (
+    <Route exact path="/">
```

Comment:
This Route wrapper is present in App component. Should work fine if you remove it here.